cartoterra/static/css/extra.css

```diff
@@ -300,6 +300,19 @@ body:before {
     width: 100%;
 }
 
+.cartoterra-home-categories {
+    margin-left:-25px;
+}
+
+.cartoterra-home-categories-legend {
+    margin-top:6px;
+    font-weight:bold;
+}
+
+.cartoterra-home-welcome-cont {
+    width: 338px;
```

Comment:
added because image should be fixed width. As image width<smaller devices width, it remains usable. categories col-classes were adapted for optimal viewing according to device width